sdk/examples/wallet/09_mint_native_token.rs

```diff
@@ -52,6 +50,10 @@ async fn main() -> Result<()> {
     };
 
     let transaction = account.mint_native_token(native_token_options, None).await?;
+
+    // Ensure the account is synced after minting.
```

Comment:
`mint_native_token()` only creates the transaction, but it doesn't wait for it to be confirmed, so if the tx isn't confirmed yet, this is useless
One could add `account.retry_transaction_until_included(transaction.transaction_id, None, None).await?;` to wait for the confirmation (also tries to get it confirmed)

Reply:
Yes, actually for all functions that create a transaction